packages/application-extension/schema/menus.json

```diff
@@ -13,6 +13,13 @@
           {
             "command": "notebook:trust",
             "rank": 20
+          },
+          {
+            "type": "submenu",
```

Comment:
Not sure it should be `submenu` here, since the entry appears as a regular single entry?

Also when trying this PR on Binder the menu entry still seems to be displayed:

![image](https://user-images.githubusercontent.com/591645/233287665-3f0664b6-5554-46ad-a2d3-a0f506dda0c4.png)